tensorflow/lite/tools/evaluation/tasks/coco_object_detection/BUILD

```diff
@@ -43,7 +43,7 @@ cc_binary(
     copts = tflite_copts(),
     linkopts = task_linkopts(),
     deps = [
-        ":run_eval_lib",
         "//tensorflow/lite/tools/evaluation/tasks:task_executor_main",
+        ":run_eval_lib",
```

Comment:
Thx for catching this compilation issue! I think a proper fix is to put "alwayslink = 1," to run_eval_lib rule. Could you give it a try? We have tried it on our side, and it worked.

Reply:
Yes, `alwayslink = 1` works for both Android and linux. I'll close this.